argon2/src/lib.rs

```diff
@@ -64,9 +64,6 @@
 //! [Password Hashing Competition]: https://www.password-hashing.net/
 
 #![no_std]
-// TODO(tarcieri): safe parallel implementation
-// See: https://github.com/RustCrypto/password-hashes/issues/154
-#![cfg_attr(not(feature = "parallel"), forbid(unsafe_code))]
```

Comment:
If you've successfully removed `unsafe`, this can be changed back to:

```rust
#![forbid(unsafe_code)]
```

Reply:
Well, spoke too soon, I see the `unsafe` code is now in `memory_view` and appears to be mandatory.